pubspec.yaml

```diff
@@ -3,7 +3,7 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 8.0.0-dev.3
+version: 8.0.0
```

Comment:
Suggest changing the SDK constraint to `sdk: ">=3.0.0-417 <4.0.0"`

Reply:
done